build/config/android/config.gni

```diff
@@ -54,10 +54,8 @@ if (is_android) {
 
   # Defines the name the Android build gives to the current host CPU
   # architecture, which is different than the names GN uses.
-  if (host_cpu == "x64") {
+  if (host_cpu == "x64" || host_cpu == "x86") {
```

Comment:
This means that build should always be done on 64-bit host, right, while the removed code theoretically allowed you to build on 32-bit  host?

Reply:
In theory, yes. In practice the 32-bit toolchain doesn't actually work, and we don't have any 32-bit-only hosts.

Reply:
okay, sgtm